static/assets/views/entities.html

```diff
@@ -1,28 +1,28 @@
 
-<h1>{{ctrl.title}} Entity Reference</h1>
+<h1>{{ctrl.apiTitle}} Entity Reference</h1>
 <p>Updated: {{ctrl.lastUpdatedDate}}</p>
 
 <p>
-    This topic describes the entity and type representations for common items that the {{ctrl.title}} API uses. To learn
+    This topic describes the entity and type representations for common items that the {{ctrl.apiTitle}} API uses. To learn
     about how to set up your developer account and work with the SKY API, see the
     <a ng-href="{{ ctrl.getStartedUrl }}">Getting Started</a> guide.
 </p>
 
 <p ng-show="ctrl.showErrorMessage" class="alert alert-danger">
-    There was an issue loading the Entity Reference for the {{ctrl.title}} API.
+    There was an issue loading the Entity Reference for the {{ctrl.apiTitle}} API.
 </p>
 
 <ul id="entities">
     <li ng-repeat="entity in ctrl.entities" style="margin: 0px">
-        <a ng-href="#{{entity.name}}">{{entity.name}}</a>
+        <a ng-href="#{{entity.name}}">{{entity.displayName}}</a>
     </li>
 </ul>
 
 <div ng-repeat="entity in ctrl.entities">
     <a ng-attr-name="{{entity.name}}"></a>
     <p>&nbsp;</p>
-    <h2>{{entity.name}}</h2>
-    <p>The <strong>{{entity.name}}</strong> entity has the following properties:</p>
+    <h2>{{entity.displayName}}</h2>
+    <p>The <strong>{{entity.displayName | lowercase}}</strong> entity has the following properties:</p>
```

Comment:
I'm not familiar with this syntax, but I'm assuming this takes the displayName and makes it lowercase. That's going to eventually cause problems when you have some acronym or proper noun for the entity display name. Probably not a priority to fix now, but something that will come up later. I think we could have entities include an optional `x-display-name-sentence-case` that we use to coalesce with the current algorithm.